### PR TITLE
Apply -Wunusual-code to spdif_tx_reconfig_port()

### PR DESCRIPTION
**Pull request by xross**

### CODE REVIEW

File: lib_spdif/src/SpdifTransmitReconfigPort.xc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#include <xs1.h>
+#include "spdif.h"
+
+/* Note, this function is in a seperate fill such that -Wunusual-code can be applied to it */
```
Comment (ed-xmos) [inline lib_spdif/src/SpdifTransmitReconfigPort.xc:7]: typo - fill -> file

**Review by ed-xmos — CHANGES_REQUESTED** [review]
BOOM! Happy to see this. Maybe a changelog entry makes sense?
FIXED - warning about legal use of resource pointe

**Comment by xross** [comment]
> BOOM! Happy to see this. Maybe a changelog entry makes sense? FIXED - warning about legal use of resource pointe



> BOOM! Happy to see this. Maybe a changelog entry makes sense? FIXED - warning about legal use of resource pointe

Done :)

**Review by ed-xmos — APPROVED** [review]
Great!

**Pull request merged**